docs: reformat the documentation (#396)
ansys/ansys-sphinx-theme: doc/source/examples/admonitions.rst

.. _admonitions:

Admonitions
===========

> Admonitions are a way to highlight important information in documentation.

Reviewer: ```suggestion
Admonitions are specially formatted blocks that highlight important information in the documentation.
```

> This page shows how the admonitions are rendered in the documentation with ``ansys sphinx theme``.

Reviewer: ```suggestion
This page shows how the Ansys Sphinx Theme renders admonitions.
```

.. jinja:: admonitions

    {% for filename in inputs_admonitions %}

    .. include:: sphinx_examples/{{ filename }}

    {% endfor %}

> Credits and copyright

Reviewer: Does this section and child section need to exist. It seems that the information can be provided as part of the Admonitions section, without headings?

Author: sure, we can add that

---------------------

The examples shown above are derived from the PyData Sphinx Theme project, copyrighted by pandas, 2018.

> Original examples can be found at `PyData sphinx theme examples <pydata_examples_>`_.

Reviewer: ```suggestion
Original examples can be found in `Examples <pydata_examples_>`_ in
the PyData Sphinx Theme documentation.
```

These examples are licensed under the BSD 3-Clause License.

Original copyright
~~~~~~~~~~~~~~~~~~

> The examples included here were originally copyrighted and licensed as follows:

Reviewer: The "included here" is referring to the examples in the PyData Sphinx Theme documentation or in the Ansys Sphinx Theme documentation? Again, I'm not sure why this can't be a paragraph after the examples rather than in multiple sections.

Author: examples included in ansys sphinx theme this page.

Author: added as `The examples included below in this page were originally copyrighted and licensed as follows:`

Author: and moved this sections on the sentence along with admonitions

- Copyright (c) 2021, Pradyun Gedam. Licensed under the Creative Commons Attribution-ShareAlike 4.0 International License

>   (SPDX-License-Identifier: CC-BY-SA-4.0), as specified by PyData Sphinx theme in their examples.

Reviewer: ```suggestion
  (SPDX-License-Identifier: CC-BY-SA-4.0), as specified by the PyData Sphinx Theme in their examples.
```
